src/main/java/pinger/Pinger.java

```diff
@@ -40,7 +40,7 @@ public static void main(String[] args)  {
                     throw new Exception("Not a http url");
                 }
             } catch (Exception e) {
-                System.out.println(e);
+                e.printStackTrace();
```

Comment:
This is basically what declaring the exception to be thrown did. Please print out a message saying that an error occurred too.